### ISSUE TITLE
MOTIS: Fix crash

### ISSUE THREAD

**Issue by jbruechert** [public-transport/transitous#105]
```
Mär 07 19:54:20 vm-MOTIS motis[26967]: 2024-03-07T18:54:20Z [VERIFY FAIL] no first valid found: ICE 77 (dbg=ch_opentransportdataswiss.gtfs.zip/stop_times.txt:75467:75469)
Mär 07 19:54:20 vm-MOTIS motis[26967]: terminate called after throwing an instance of 'std::runtime_error'
Mär 07 19:54:20 vm-MOTIS motis[26967]:   what():  no first valid found: ICE 77 (dbg=ch_opentransportdataswiss.gtfs.zip/stop_times.txt:75467:75469)
```

**Comment by felixguendling** [comment]
Yes, I updated the GTFS dataset from 2023 to 2024 to be able to reproduce the issue. Now writing a test case with this specific trip and GTFS RT update to be able to debug it faster.

**Comment by felixguendling** [comment]
They set all three stops to `SKIP` which is effectively a cancellation of the whole train without setting the canceled flag. I'll add a check for this case and set those trips to canceled.

**Comment by felixguendling** [comment]
Should be fixed with [v0.11.25](https://github.com/motis-project/motis/releases/tag/v0.11.25). Please let me know if you encounter any problems. It also contains a performance fix for the footpaths preprocessing problem (see https://github.com/motis-project/nigiri/pull/76 for my measurements).
